src/lib/y2storage/boot_requirements_strategies/base.rb

```diff
@@ -213,10 +213,15 @@ def unknown_boot_disk_error
         SetupError.new(message: error_message)
       end
 
-      # Whether the boot device is encrypted and grub can decrypt it
+      # Whether the boot device can be read by grub
       #
-      # @return [Boolean] true if grub can decrypt boot device (or it is unencrypted)
-      def encrypted_for_grub?
+      # The boot device can be read by grub when:
+      #
+      # * it is not encrypted (obviously),
+      # * or it is encrypted using LUKS1.
+      #
+      # @return [Boolean] true if grub can read decrypt boot device (or it is unencrypted)
+      def readable_grub?
```

Comment:
I know is a bit too long but, what about something like `#boot_readable_by_grub?`? My point is this is not about reading grub, but about grub reading boot

Reply:
Or maybe `#readable_kernel?`... I don't know